AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -915,16 +915,17 @@ function Request-AzPowerShellModule {
                             -ErrorAction SilentlyContinue
                 } catch {
                     Write-Error `
-                            -Message "Unable to uninstall the GA version of the Az.Storage module in favor of the preview version (1.11.1-preview)." `
+                            -Message "Unable to uninstall the pre-GA version of the Az.Storage module in favor of the GA version (2.0.0)." `

```

Comment:
Nit: adjust error message to handle future versions and maybe we should change "GA" to "release" or "general availability" in case customers don't know GA term.